ui/.eslintrc.json

```diff
@@ -0,0 +1,13 @@
+{
+  "extends": ["eslint:recommended", "next/core-web-vitals", "prettier"],
+  "rules": {
+    "@next/next/no-img-element": "off"
```

Comment:
We sure we want to disable this rather than using next.js image components? They bring a lot of potential benefits for speed.

Reply:
The `<Image />` component doesn't support `next export` (which is how we bundle the UI). That said, we should consider this if we do consider allowing the UI to run as an external process